packages/drag-drop/demo/~patterns/stories/SortableListsStory.tsx

```diff
@@ -0,0 +1,400 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { RefObject, forwardRef, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+import type { Story } from '@storybook/react';
+import type {
+  Announcements,
+  DragEndEvent,
+  DragOverEvent,
+  DragStartEvent,
+  UniqueIdentifier
+} from '@dnd-kit/core';
+
+import {
+  MeasuringStrategy,
+  closestCorners,
+  DragOverlay,
+  DndContext,
+  KeyboardSensor,
+  MouseSensor,
+  TouchSensor,
+  useDroppable,
+  useSensor,
+  useSensors
+} from '@dnd-kit/core';
+import {
+  sortableKeyboardCoordinates,
+  SortableContext,
+  verticalListSortingStrategy,
+  arrayMove,
+  useSortable,
+  AnimateLayoutChanges,
+  defaultAnimateLayoutChanges
+} from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+import {
+  Draggable,
+  DraggableList,
+  Dropzone,
+  IDraggableProps
+} from '@zendeskgarden/react-drag-drop';
+
+interface IColumnProps {
+  id: UniqueIdentifier;
+  items: ISortableItem[];
+}
+
+interface IDropzoneColumnProps extends IColumnProps {
+  activeId: UniqueIdentifier | null;
+  activeColumnId: UniqueIdentifier | null;
+  activeItem?: ISortableItem | null;
+}
+
+interface ISortableItem {
+  label: string;
+  caption: string;
+  id: UniqueIdentifier;
+}
+
+interface ISortableItemProps extends IDraggableProps {
+  data: ISortableItem;
+  showDropmessage?: boolean;
```

Comment:
#nit - `s/showDropmessage/showDropMessage` maybe use camel case for readability?